config.yml

```diff
@@ -14,6 +15,28 @@ ska:
   fastq_qual: 20
   fastq_cov: 4
 
+# gubbins options
+gubbins:
+  prefix: gubbins
+  tree-builder: iqtree
+  min-snps: 3
+  min-window-size: 100
+  max-window-size: 10000
+  iterations: 10
+  dir: output/strains/{strain}/
+
+# transphylo options
+transphylo:
+  w_shape: 10
+  w_scale: 0.1
+  mcmcIterations: 1000
+  startNeg: 0.2739726 #100/365
+  startOff_r: 1
+  startOff_p: 0.5
+  startPi: 0.5
+  optiStart: 2
+  dateT: 2008.01
```

Comment:
Why this value?

Reply:
I wanted to set it to `max(dates)` but then I always get this error: `The parameter dateT cannot be smaller than the date of last sample`.
So then I assumed it had to be just a tiny bit larger than that